public/js/log.js

```diff
@@ -60,250 +59,295 @@ MAGI_log.resizes = {
 };
 MAGI_log.tooltips = [];
 
-$('#magi-loggingReadMore').click(function(e) {
-  e.preventDefault();
-  var isVisible = 'visible' == $('#magi-loggingReadMoreDiv').css('visibility'),
-      visibleState = isVisible ? 'hidden' : 'visible',
-      displayState = isVisible ? 'none' : 'block';
-
-  $('#magi-loggingReadMoreDiv').css('visibility', visibleState);
-  $('#magi-loggingReadMoreDiv').css('display', displayState);
-
+$().ready(function () {
+  $.get('/logEnabled', function(res, status) {
```

Comment:
Not sure I get why you want to determine if logging is enabled by checking with the server. Wouldn't it be easier to just pass in a variable from the server?